doozer/doozerlib/cli/scan_osh.py

```diff
@@ -0,0 +1,114 @@
+import click
+import koji
+from doozerlib.cli import cli, click_coroutine, pass_runtime
+from doozerlib.runtime import Runtime
+from doozerlib.exectools import fire_and_forget
+from doozerlib.constants import BREWHUB_URL
+from doozerlib.util import cprint
+
+
+class ScanOshCli:
+    def __init__(self, runtime: Runtime, brew_tags: list, last_brew_event):
+        self.runtime = runtime
+        self.brew_tags = brew_tags
+        self.last_brew_event = last_brew_event
+
+        # Initialize runtime
+        self.runtime.initialize(clone_distgits=False, no_group=True)
+
+    @staticmethod
+    async def get_tagged_latest(tag):
+        """
+        Returns the latest RPMs and builds tagged in to the candidate tag received as input
+        """
+        session = koji.ClientSession(BREWHUB_URL)
+        try:
+            latest_tagged = session.listTagged(tag=tag, latest=True)
+            if latest_tagged:
+                return latest_tagged
+        except Exception:
+            raise
+
+    @staticmethod
+    async def get_tagged_all(tag):
+        """
+        Returns all the RPMs and builds that are currently in the candidate tag received as input
+        """
+        session = koji.ClientSession(BREWHUB_URL)
+        try:
+            latest_tagged = session.listTagged(tag=tag, latest=False)
+            if latest_tagged:
+                return latest_tagged
+        except Exception:
```

Comment:
This is a no-op, unless you do something in the exception handler. Otherwise, you might just as well remove the try-except

Reply:
fixed in https://github.com/openshift-eng/art-tools/pull/51/commits/906d84751f15b00b02083b9cfe428b05c0888896